cfgov/jinja2/v1/_includes/molecules/image-inset.html

```diff
@@ -0,0 +1,74 @@
+{# ==========================================================================
+
+    Image Inset
+
+    ==========================================================================
+
+    Description:
+
+    Creates image inset markup when given:
+
+    value:                   Object defined from a StreamField block.
+
+    value.heading:           The heading of the image inset.
+
+    value.image:             A basic image object to insert.
+
+    value.image.upload:      The image object.
+
+    value.image.alt:         String for alt text for screenreaders.
+
+    value.paragraph:         String for text below the image.
+
+    value.is_image_left:     Boolean indicating whether to float the image
+                             to the left.
+
+    value.image_width:       String representing the image width.
+                             Possible values: (150, 270, 300).
+                             Defaults to 270.
+
```

Comment:
Fixed in https://github.com/cfpb/cfgov-refresh/pull/2678/commits/98cde738538340d61bc985dcc16f0bdb95da36c9.